client/components/app.jsx

```diff
@@ -40,6 +40,9 @@ export default class App extends React.Component {
             <Route path="/menu-customizer">
               <MenuCustomizer />
             </Route>
+            <Route path="/sales-report">
```

Comment:
jason already re-added this.  I'll approve but make sure you test it to make sure it isn't broken